spec/controllers/api/v1/holidays_controller_spec.rb

```diff
@@ -34,7 +34,6 @@
       it "Should not include current day if it is a holiday", :show_in_doc do
         create(:holiday, holiday: Time.now.beginning_of_day)
         get :available_dates, schedule_days: 6
-        pp response.body
```

Comment:
yay! thanks for finding that.. I'd been searching for 'puts' and 'print' etc etc but couldn't locate it!

Reply:
I could take credit for removing it, but i'm also the one who added it 🤣